### PR TITLE
Remove unused includes/imports and re-order alphabetically

### PR DESCRIPTION
**Pull request by ziggyman**
the remaining ones.

### CODE REVIEW

File: config/flat.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import os.path
 from lsst.utils import getPackageDir
+import os.path
```
Comment (RobertLuptonTheGood) [inline config/flat.py]: System imports go before LSST/PFS ones.  Please check all changesets and fix (and squash)

File: bin.src/genDefectFits.py [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-import os.path
-import re
-
-import numpy
-import pyfits
 import collections
-
 import lsst.log as log
 from lsst.obs.pfs import PfsMapper
+import numpy
```
Comment (RobertLuptonTheGood) [inline bin.src/genDefectFits.py]: Elsewhere you import numpy as np.  Please file a ticket to consistently use `import numpy as np` everywhere.

File: config/flat.py (hunk 1 above)
Comment (ziggyman) [inline config/flat.py]: Done. I did look at the style guide but did not find anything about system imports going before LSST/PFS ones...

File: bin.src/genDefectFits.py (hunk 2 above)
Comment (ziggyman) [inline bin.src/genDefectFits.py]: Done. Filed ticket PIPE2D-210

**Pull request merged**